app/classifier/tasks/generic-editor.cjsx

```diff
@@ -47,10 +47,12 @@ module.exports = React.createClass
           <small className="form-help">Add text and images for a window that pops up when volunteers click “Need some help?” You can use markdown to format this text and add images. The help text can be as long as you need, but you should try to keep it simple and avoid jargon.</small>
         </div>}
 
-      <hr />
+      {if choicesKey?
+        <hr />
 
-      <span className="form-label">Choices</span>
-      {' '}
+        <span className="form-label">Choices</span>
```

Comment:
Little JSX gotcha: an CoffeeScript `if` expression like:

```jsx
{if true
  <OneThing />
  <Another />}
```

can only resolve to a single value, so this becomes `<Another />`, where `<OneThing />` is skipped.

This particular file works around the problem with the convention of resolving to an array, (and each item has an arbitrary `key` property), but that's probably a little convoluted. I'd just wrap these in a container `<div>` so they'll both display.